docs/configuration/miscellaneous.md

```diff
@@ -180,6 +180,14 @@ When determining the primary IP address for a device, IPv6 is preferred over IPv
 
 ---
 
+## PUBLIC_SWAGGER
```

Comment:
I'm going to lightly suggest flipping this to `SWAGGER_PUBLIC`, so that it would fit better with potential future configuration options for Swagger.

Reply:
ack, done in the new commit.